resistbot-node-backend/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import { handleIncomingDM, handleQuickResponse, handleRapidProResponse } from './dmHandlers';
+import logger from 'morgan';
+
+var app = express();
+const port = process.env.PORT || 5555;
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+/* GET home page. */
+app.get('/', function(req: express.Request, res: express.Response) {
+  res.send('<h1>Hello from Node & Docker</h1>');
+});
+
```

Comment:
Are these paths something we can make whatever we want? Just want to make sure I understand what things we control or not.

Reply:
These paths can be whatever we want. Architecturally, events come into the gateway-forwarder via websockets, and the forwarder then sends the messages along to the endpoint via HTTP requests.

The original reason for this architecture was so that we could spin up many of the actual handlers, while having a single websocket listener that's pretty fast to offload the work. That may be overkill for now, and an 80/20 would be to just merge the two node apps. The one strictly necessary endpoint is the one that rapidpro hits to send messages to the user, and I think that's configured on the rapidpro side.

In addition, I added husky, lint-staged, and a default installation of prettier. Skipped eslint for now because it got late!